model-training/train_ctc_model.py

```diff
@@ -8,7 +8,9 @@
 
```

Comment:
Have you run train_ctc_model.py with all configs you've pushed? i.e., quatznet_15x5, conformer_ctc_bpe, and conformer_ctc_char? I see that you show only conformer_ctc_char in your medium article.

Also, can you tell me whether any of these configs correspond to the model we used for The People's Speech paper?

Reply:
The training config for the paper is `conformer_ctc_char`. Been a while since I ran the other two.

Reply:
Do you believe that conformer_ctc_bpe and quartznet_15x5 will work when ran? If you have any doubt, we should remove them. That way we have less code to "support".

Reply:
Removed and pushed